integration-tests/cypress/e2e/matching.feature

```diff
@@ -112,7 +112,7 @@ Feature: Matching defendants to nDelius records
       | Previously known                                              |
       | 13 January 2015 - CJA Standard Determinate Custody (6 Months) |
     And I should see radio buttons with the following IDs
-      | defendant-1 | defendant-2 | defendant-3 | defendant-4 | defendant-5 |
+      | defendant-1 | defendant-2 | defendant-3 | defendant-4 | defendant-5 |defendant-5
```

Comment:
How can there be two radio buttons with the same id `defendant-5`?

ids have to be unique in the DOM

Reply:
agreed, as soon as i raised it - realised and have made the changes

Reply:
That's a great catch